src/libs/PolicyUtils.ts

```diff
@@ -13,10 +13,10 @@ type UnitRate = {rate: number};
  * Filter out the active policies, which will exclude policies with pending deletion
  * These are policies that we can use to create reports with in NewDot.
  */
-function getActivePolicies(policies: OnyxCollection<Policy>): Policy[] | undefined {
-    return Object.values(policies ?? {}).filter<Policy>(
+function getActivePolicies(policies: OnyxCollection<Policy>): Policy[] {
+    return Object.values(policies ?? {}).filter(
         (policy): policy is Policy =>
-            policy !== null && policy && (policy.isPolicyExpenseChatEnabled || policy.areChatRoomsEnabled) && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
+            policy !== null && !!policy && (!!policy.isPolicyExpenseChatEnabled || !!policy.areChatRoomsEnabled) && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
```

Comment:
```suggestion
           (!!policy?.isPolicyExpenseChatEnabled || !!policy?.areChatRoomsEnabled) && policy?.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
```
wdyt?

Reply:
That does seem more readable and should protect from both null and undefined. Thanks!